package.json

```diff
@@ -34,6 +34,6 @@
     "xml": "^1.0.0"
   },
   "peerDependencies": {
-    "mocha": "^2.2.5"
+    "mocha": "^3 || ^2.2.5"
```

Comment:
@michaelleeallen @clayreimann It's the normal way of declaring a version range with carets, although it could also be written as `>=2.2.5 <4` if you prefer one of the [other syntaxes](https://docs.npmjs.com/misc/semver).

`>=2.2.5` means you're declaring anything above is supported. If you want to go with that it's completely fine by me, I just assumed since you were using caret ranges that you'd want to declare `^3` support without quite yet declaring you were also supporting `>=4`?

Let me know which range you'd like and I'll update the PR 😉  (Also, would you prefer a second commit, or a rebase?)